src/main/resources/reference.conf

```diff
@@ -89,4 +91,30 @@ kamon {
     }
   }
 
-}
\ No newline at end of file
+  modules {
+    datadog-agent {
+      enabled = true
+      name = "DatadogAgent"
+      description = "Datadog agent reporter"
+      class = "kamon.datadog.DatadogAgentReporter"
+      factory = "kamon.datadog.DatadogAgentReporterFactory"
+    }
+
+    datadog-api {
+      enabled = true
```

Comment:
Probably this one should be disabled by default since it would be reporting the same data that the agent one is reporting.